src/functions.rs

```diff
@@ -293,21 +569,23 @@ fn col(name: &str) -> PyResult<PyExpr> {
     })
 }
 
+// TODO: should we just expose this in python?
 /// Create a COUNT(1) aggregate expression
 #[pyfunction]
-fn count_star() -> PyResult<PyExpr> {
-    Ok(PyExpr {
-        expr: Expr::AggregateFunction(AggregateFunction {
-            func_def: datafusion_expr::expr::AggregateFunctionDefinition::BuiltIn(
-                aggregate_function::AggregateFunction::Count,
-            ),
-            args: vec![lit(1)],
-            distinct: false,
-            filter: None,
-            order_by: None,
-            null_treatment: None,
-        }),
-    })
+fn count_star() -> PyExpr {
+    functions_aggregate::expr_fn::count(lit(1)).into()
+}
+
```

Comment:
Per the question:

I think it's worth adding some guidance about what to put in python vs what to put in rust for this repo. In my mind the things that should go into the python side are
- Trivial aliases (this is a good example)
- Simple type conversion, like path -> string of the path or number to lit(number)
- More complex type conversion *if* it makes sense to do in python. For example, in the `named_struct` where sending in a dictionary on the python side makes a lot more sense and isn't as simple to do via pyo3.

And then I'd lean towards everything else sitting on the rust side.

Reply:
Should I add that to the `functions.rs` module docs and/or somewhere in the contributor guide?

Reply:
I'd say the contributor guide since it's more general guidance and not specific to functions